### ISSUE TITLE
custom Sound not play in Android 9

### ISSUE THREAD

**Issue by Osamasomy** [zo0r/react-native-push-notification#1400]
# Bug

Im Very thankful to you for this liabrary.I integrate this liabrary with my project all works fine but then I test on Android 9 the custom sound is not playing(no sound) and my other device which have android 7 all things are working as expected.

I think issue relate to API level 9 or Higher then 7.

## Environment info

`react-native info` output:

```bash
 System:
    OS: Windows 10 10.0.18362
    CPU: (4) x64 Intel(R) Core(TM) i3-2370M CPU @ 2.40GHz
    Memory: 1.34 GB / 7.97 GB
  Binaries:
    Node: 10.17.0 - C:\Program Files\nodejs\node.EXE
    npm: 6.11.3 - C:\Program Files\nodejs\npm.CMD
  SDKs:
    Android SDK:
      API Levels: 23, 24, 25, 28, 29
      Build Tools: 28.0.3, 29.0.2
      System Images: android-28 | Google Play Intel x86 Atom
  npmPackages:
    react: 16.9.0 => 16.9.0
    react-native: 0.61.5 => 0.61.5
    react-native-push-notification: "^3.1.9",
```

Library version: 3.1.9

## Steps To Reproduce
1. Just configure this liabrary and place custom sound file in /res/raw
2. Try to Test on Android 9
...

Describe what you expected to happen:

1. I just Expected Its behave same on All devices as its working on android 7

## Reproducible sample code
```
PushNotification.localNotificationSchedule({
        title: `Test`,
        soundName: "sound.mp3", // **Here Is the Problem in Android 9**
        message: "test message", // (required)
        date: (new Date(Date.now() + ( ( 60 * 1000)))),
      });
```

**Comment by Dallas62** [comment]
Hi @Osamasomy 
I will suggest you to test the version [`3.3.1`](https://github.com/zo0r/react-native-push-notification/releases/tag/3.3.1)
You might need to clean build cache/node_modules and reinstall the application.
Regards,

**Comment by Dallas62** [comment]
Any updates ?

**Comment by Osamasomy** [comment]
still not working on android 9 no sound play on android 9 and works fine on 7.

**Comment by kdpadhiyar** [comment]
https://github.com/zo0r/react-native-push-notification/issues/1423#issue-614600690

lol his own example is not plying sound

**Comment by Dallas62** [comment]
Again @kdpadhiyar , If this make you laugh, feel free to make a PR or a working library :)

**Comment by kdpadhiyar** [comment]
Well if i find solution i will make noise same way .
I am trying it from two days to solve it.

**Comment by Dallas62** [comment]
Hi @Osamasomy 
I will put an update on the example project, but it's working fine in `3.3.1`. Maybe it's an issue in configuration.

**Comment by Dallas62** [comment]
The example project is now available.
If you still encounter this issue, please reopen the issue with a reproducible code.

**Comment by u15767** [comment]
Hi, this really doesn't work. It is not a matter of custom sound. I debugged the code, especially the RNPushNotificationHelper.java

All notifications with default settings will be notified in a channel with the id
`rn-push-notification-channel-id-4-300`
This is the same id as the one generated for the default channel (see method checkOrCreateDefaultChannel), which is generated per default. Unfortunately, this channel is created without any sound. This means, any usage of notfications with all parameters set to defaults will not play sound although the default for sound is to play sound.

I don't know which solution you would prefer. I could think of providing the default sound also for the default notification channel. Another solution could be to ensure that notifications with all defaults used use a different channel.

As I don't want to decide this, I did not make a pull request out of it.
